resources/lang/es/gitscrum.php

```diff
@@ -8,184 +8,184 @@
 
 
 return[
-    'title-login' => 'GitScrum Login',
-    'welcome-to' => 'Welcome to',
-    'edit-comment' => 'Edit Comment',
-    'configuration' => 'Configuration',
-    'configuration-config-status-' => 'Configuration - Config Status - ',
+    'title-login' => 'Acceder a GitScrum',
+    'welcome-to' => 'Bienvenido a',
+    'edit-comment' => 'Editar comentario',
+    'configuration' => 'Configuración',
+    'configuration-config-status-' => 'Configuración - estatus - ',
```

Comment:
I think here status would be the rigth way. I've never seen status - estatus translation on a program before, also status is also used in spanish.